### ISSUE TITLE
binary releases don't work in Arch Linux

### ISSUE THREAD

**Issue by japaric** [japaric/xargo#62]
```
$ ./xargo -V
./xargo: /usr/lib/libcurl.so.4: version `CURL_OPENSSL_3' not found (required by ./xargo)
./xargo: /usr/lib/libssl.so.1.0.0: no version information available (required by ./xargo)
./xargo: /usr/lib/libcrypto.so.1.0.0: no version information available (required by ./xargo)
```

Tested using Xargo 0.1.14

**Comment by TomCrypto** [comment]
Temporary workaround:

1) Install `libcurl-compat` from AUR:

    yaourt -S libcurl-compat

2) Set `LD_PRELOAD` to make xargo use it:

    LD_PRELOAD=/usr/lib/libcurl.so.3 xargo ...
    # (or export LD_PRELOAD=...)

Using this in my docker image and it works, although suboptimal.

**Comment by japaric** [comment]
fixed in #65 
Xargo no longer depends on libcurl.so, etc.
